docs/flutterbloccoreconcepts.md

```diff
@@ -173,4 +173,119 @@ Since we only have one repository in our app, we will inject it into our widget
 
 Now when instantiating a bloc, we can access the instance of a repository via `context.read` and inject the repository into the bloc via constructor.
 
-[flutter_weather_link]: https://github.com/felangel/bloc/blob/master/examples/flutter_weather
\ No newline at end of file
+[flutter_weather_link]: https://github.com/felangel/bloc/blob/master/examples/flutter_weather
+
+## Extension Methods
+
+Importing the `flutter_bloc` package includes different extension methods and this section aims to describe what and how these should be used. This type of methods, included in Dart 2.7, allow to add functionalities to existing classes.
```

Comment:
```suggestion
Importing the `flutter_bloc` package includes different extension methods and this section aims to describe what they do and how they should be used. This type of method, included since Dart 2.7, allows adding functionalities to existing classes.
```